CMakeLists.txt

```diff
@@ -53,15 +53,15 @@ if(NOT CMAKE_BUILD_TYPE)
   set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS Debug Release RelWithDebInfo MinSizeRel)
 endif()
 
-option(PRECICE_MPICommunication "Enables MPI-based communication and running coupling tests." ON)
-option(PRECICE_PETScMapping "Enable use of the PETSc linear algebra library." ON)
-option(PRECICE_PythonActions "Python support" ON)
-option(PRECICE_Packages "Configure package generation." ON)
+option(PRECICE_MPI_COMMUNICATION "Enables MPI-based communication and running coupling tests." ON)
+option(PRECICE_PETSC_MAPPING "Enable use of the PETSc linear algebra library for global-iterative RBF data mappings." ON)
+option(PRECICE_PYTHON_ACTIONS "Enable Python support for preCICE actions." ON)
```

Comment:
We could also group these under the prefix `FEATURE_`, to signify to a completely new user (e.g., cluster admin), that toggling this variable toggles a feature:

```
PRECICE_FEATURE_MPI_COMMUNICATION
PRECICE_FEATURE_PETSC_MAPPING
PRECICE_FEATURE_PYTHON_ACTIONS
```

Reply:
In principle, these features are now already grouped together (for lexicographic reasons). We could go for the `_FEATURE_<feat>` naming (again would be nice for ccmake). Only counter argument is that the variables get rather lengthy.  What do you like more:

![image](https://github.com/precice/precice/assets/33414590/fdad8130-ea86-43a0-866c-e979356b558a)

![image](https://github.com/precice/precice/assets/33414590/e20a347f-f547-4e30-baa4-c3a4783c62e2)

Reply:
I would not rely to the fact that the `MPI_`, `PETSC_`, and `PYTHON_` are currently next to each other, as this can easily change in the future.

I see the issue with the truncation of names in ccmake, but this sounds like a technicality that should not concern us. ccmake could set this width arbitrarily, and more variables are already truncated.

I prefer the `FEATURE_` grouping, unless it becomes unclear to define what should belong into this group.

Reply:
Yes, checked and it does. Just in the same way as ccmake does, no?

Reply:
![image](https://github.com/precice/precice/assets/13552216/ba7b4354-01e4-4bdc-9476-56f87b3540a1)

Ok. It only groups on the first part.